epictrack-api/tests/utilities/factory_scenarios.py

```diff
@@ -31,7 +31,7 @@ class TestProjectInfo(Enum):
         "name": fake.word(),
         "description": fake.sentence(),
         "address": fake.address(),
-        "type_id": fake.random_int(min=1, max=10),
+        "type_id": fake.random_int(min=1, max=9),
```

Comment:
any particular reason for this

Reply:
there is no 10th type, when type 10 was picked Foreign key constraint error was being thrown